guides/common/modules/proc_creating-an-activation-key-to-consume-the-content-view.adoc

```diff
@@ -1,7 +1,7 @@
 [id="Creating_an_Activation_Key_to_Consume_the_Content_View_{context}"]
 = Creating an activation key to consume the content view
 
-Use the following procedure to create an activation key that you can use to subscribe your hosts to content managed by {Project}.
+Create another activation key to assign hosts to the content view you created in {ContentManagementDocURL}Creating_a_Content_View__content-management[Creating a content view] in _{ContentManagementDocTitle}_. 
```

Comment:
```suggestion
Create another activation key to assign hosts to the content view you created in {ContentManagementDocURL}Creating_a_Content_View_content-management[Creating a content view] in _{ContentManagementDocTitle}_. 
```
This _might_ even fix the linkchecker failure.

Reply:
Wait. But shouldn't this be an xref? This module is in _Managing content_, isn't it? (It's hard to check without a preview.)

Reply:
You are correct. I was going back and forth between Hosts and Content that I must have mixed them up.